### DISCUSSION TITLE
Turn off slides?

### DISCUSSION THREAD

**Discussion by barrylachapelle** [igordanchenko/yet-another-react-lightbox#44] [Q&A] [answered]
Hi there. I was wondering if there is a way to turn off the slides array and/or next/previous buttons? I only need to show one image at a time. Thank you!

Love the package by the way. I spent a full day trying to find something that worked in my project architecture well. Had yours up and running in minutes. Do you have a Buy Me A Coffee account or something? Would love to make a small donation. 🙇‍♂️

**Comment by igordanchenko** [comment, marked as answer]
Hi there!

Here is how you can hide the navigation buttons and disable swipe feature for a single slide:

```jsx
<Lightbox
  slides={[slide]}
  carousel={{ finite: true }}
  render={{ buttonPrev: () => null, buttonNext: () => null }}
```

https://codesandbox.io/s/yet-another-react-lightbox-44-rgvd1v

Is this what you were trying to achieve?

I'm glad to hear you are enjoying using this package so far. Please consider giving it a star here on GitHub in lieu of a donation.
